.github/workflows/lint-sample.yml

```diff
@@ -11,6 +11,13 @@ on:
 jobs:
   lint-samples:
```

Comment:
@mateusoliveira43 

Could you rebase this one with master and ensure that here in this file the only change is to add a new test


   ```
   - name: Check linter configuration
        working-directory: ${{ matrix.folder }}
        run: make lint-config
```

Reply:
rebased